tests/server/test_handlers.py

```diff
@@ -87,72 +85,6 @@ def test_search_runs_default_view_type(mock_get_request_message, mock_store):
     assert args[2] == ViewType.ACTIVE_ONLY
 
 
-def _assert_logged_entities(run_id, metric_entities, param_entities, tag_entities):
```

Comment:
Removed this test as it’s redundant / superseded by the LogBatch test in `test_rest_tracking`. The test_rest_tracking test is better in that it mocks fewer things (i.e. actually launches an OSS server & makes API requests against it, instead of simply calling the handler functions)